File: SMNApplication/app/src/main/java/com/vinhle/smn/model/response/LoginResponse.java
package com.vinhle.smn.model.response;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Created by VinhLe on 5/6/2017.
 */

public class LoginResponse extends BaseResponse {

    @JsonProperty("account_id")
    public int accountId;
    @JsonProperty("account_email")
    public String accountEmail;
    @JsonProperty("account_full_name")
    public String accountFullName;
    @JsonProperty("account_session")
    public String accountSession;


    @Override
    public String toString() {
        return "LoginResponse{" +
                "accountId=" + accountId +
                ", accountEmail='" + accountEmail + '\'' +
                ", accountFullName='" + accountFullName + '\'' +
                ", accountSession='" + accountSession + '\'' +
                '}';
    }
}

File: SMNIOS/SMNApp/SMNApp/API/Model/Response/LoginResponse.swift
//
//  LoginResponse.swift
//  SMNApp
//
//  Created by Nguyen Thuan Tan on 6/26/17.
//  Copyright © 2017 Nguyen Thuan Tan. All rights reserved.
//

import Foundation
import ObjectMapper

public class LoginResponse: BaseResponse {
    public var accountId:Int!
    public var accountEmail:String!
    public var accountFullName:String!
    public var accountSession:String!
    
    public required init?(map: Map) {
        super.init(map: map)
    }
    
    override public func mapping(map: Map) {
        super.mapping(map: map)
        
        accountId <- map["account_id"]
        accountEmail <- map["account_email"]
        accountFullName <- map["account_full_name"]
        accountSession <- map["account_session"]
    }
}
